src/routes/workshops.js

```diff
@@ -1,9 +1,13 @@
 const express = require("express");
+const fs = require("fs");
 
 const router = express.Router();
 
 router.get("/workshops", (req, res) => {
-  res.render("workshops");
+  fs.readFile("data/workshops.json", (err, data) => {
```

Comment:
Can you please do it the same way like in Courses and Home routers?

Reply:
@pheeria Done